util/rai_node.spec

```diff
@@ -0,0 +1,47 @@
+Name:                   rai_node
+Version:                16.0.0
+Release:                1%{?dist}
+Summary:                RPM packaged nano daemon
+License:                BSD-2-Clause
+URL:                    https://nano.org
+Requires(pre):          /usr/sbin/useradd, /usr/bin/getent
+Requires(postun):       /usr/sbin/userdel
+
+%description
+This is the RPM based installer for the nano daemon, handling
+installation of the nano daemon to /usr/bin, creating a
+service unit and user nanocurrency in /var/nanocurrency
+
+%prep
+rm -fr *
```

Comment:
This will all go away when we move to including the spec file in the release tarball;  I think the "cmake" command should go in the `%build` section, which seems to be typical